scripts/devops_tasks/tox_harness.py

```diff
@@ -208,9 +207,11 @@ def inject_custom_reqs(file, injected_packages, package_dir):
         )
         with open(file, "r") as f:
             for line in f:
+                logging.info("Attempting to parse {}".format(line))
                 try:
                     parsed_req = [req for req in parse_requirements(line)]
-                except RequirementParseError as e:
+                except Exception as e:
+                    logging.error(e)
```

Comment:
If we can't parse it, it's going to be a valid non-pypi requirement.

EG: `git vcs` or `raw path` or `external URI reference`

I'm pretty comfortable just letting the issues ride here due to the fact that this runs within CI only. (`TF_BUILD=true` only in CI unless devs set it manually)

The requirements we are getting out of this thing are correct.